README.md

```diff
@@ -587,6 +588,15 @@ java -jar modules/swagger-codegen-cli/target/swagger-codegen-cli.jar generate \
   -o samples/server/petstore/haskell-servant
 ```
 
+### ASP.NET 5 Web API
+
+```
+java -jar modules/swagger-codegen-cli/target/swagger-codegen-cli.jar generate \
+  -i http://petstore.swagger.io/v2/swagger.json \
+  -l aspnet5-webapi \
```

Comment:
I'll leave the header as `ASP.NET 5 Web API`. Technically, MVC and Web API are being merged and this is why I renamed from `aspnet5-webapi` to just `aspnet5`. I think the header will make it clear for now that there aren't any server-side views, and that the generator focuses only on the api.